### DISCUSSION TITLE
Is there a way disable dashboard capability?

### DISCUSSION THREAD

**Discussion by sudhakar-datta** [netdata/netdata#17122] [Q&A] [answered]
We are not using the dashboard feature, instead we do remote write. Is there a way to disable the feature?

**Comment by sudhakar-datta** [comment, marked as answer]
https://learn.netdata.cloud/docs/netdata-agent/securing-netdata-agents#disable-the-local-dashboard

**Reply by tkatsoulas** [reply]
yes!
